content/posts/newsletter-020/index.md

```diff
@@ -1060,14 +1036,14 @@ Version 0.5 is hot off the presses and rolls up features from the 0.4
 development cycle:
 
 - Derive macros can now implement `PartialEq` and `PartialOrd` between
-  archived and unarchived types
+  archived and unarchived types.
 - Custom type bounds for serialization and deserialization can be added
-  with derive attributes
+  with derive attributes.
 - Helper types like [AlignedVec][rkyv-AlignedVec] and
-  [Infallible][rkyv-Infallible] were introduced to improve ergonomics
-- `const_generics` are now enabled by default
-- Helper functions have been added to make getting root objects easier
-- Several bugfixes and performance improvements
+  [Infallible][rkyv-Infallible] were introduced to improve ergonomics.
```

Comment:
were => was

Reply:
:thinking: Are you sure? "Helper **types** like AlignedVec and Infallible **were** introduced to improve ergonomics"

Reply:
My bad, there's nothing to fix here. In my morning grog I missed the previous markdown line.